scene/resources/style_box.cpp

```diff
@@ -105,6 +105,7 @@ void StyleBoxTexture::set_texture(RES p_texture) {
 	texture = p_texture;
 	emit_signal("texture_changed");
```

Comment:
`StyleBoxTexture` had a `texture_changed` signal already, but it was never emitted, rendering it useless. I think the `texture_changed` signal is already redundant, as `Resource` has a `changed` signal, but I added it to maintain consistency; otherwise, I would have opted to remove it.